task-5/task-5/src/timer.ts

```diff
@@ -0,0 +1,165 @@
+export class Timer {
+  timer
+  isRunning: boolean
+  type
+  time: number
+  timeElement
+  startBtn
+  stopBtn
+  resetBtn
+  typesButton
+  focusBtn
+  shortBtn
+  longBtn
+
+  constructor(time: number) {
```

Comment:
why do you take time for Pomodoro timer, pomodoro has prefixed times for tasks duration, breaks, etc..

Reply:
I forgot to remove, all times are predefined in methods.